lib/commons/dom/is-hidden-with-css.js

```diff
@@ -10,6 +10,14 @@
  * @return {Boolean} the element's hidden status
  */
 dom.isHiddenWithCSS = function isHiddenWithCSS(el, descendentVisibilityValue) {
+	if (el._isHiddenWithCSS === void 0) {
```

Comment:
So `el` is an HTMLElement, which we try not to add properties to as it can be unreliable. Instead, you should get the VirtualNode from the element and put the cache on that.

```js
const vNode = axe.utils.getNodeFromTree(el);

if (vNode._isHiddenWithCss === void 0) {

}
```